### PR TITLE
Nerf Frag Grenades & Cyborg self-destruct

### PR DESCRIPTION
**Pull request by LordFowl**
Frag grenades will now never cause a hole to space, although they will cause more superficial damage, retaining their lethality.

Cyborgs, when self-destructing, will now lockdown for five seconds, announce to everyone around them that they are self-destructing, and then explode (Again, never causing a hole to space), and launch 50-100 shrapnel (1/4th - 1/2th the amount of a frag grenade. Each shrapnel is 1/3rd as damageful, and steps half as far.)

Chelms (Golem brains) can no longer be put into MMIs. Never made any sense.

### CODE REVIEW

**Comment by NanakoAC** [comment]
i dont see why exploding cyborg shouldn't cause a hull breach, they're pretty big. IMO either the warning beforehand, OR no space breaching, both seems a bit too weak

**Comment by skull132** [comment]
Hill breach is too much. They don't have enough explosives in them to shred the hull without you weakening the hull first.

**Comment by LordFowl** [comment]
The objective of the PR is _not_ to kill people, or really even technically make cyborgs any more dangerous when exploding. It is merely to add a little more flavour. However, do note that if you _do_ manage to get caught in the explosion, you're still looking at sustaining 60-120 brute damage, as well as receiving embedded shrapnel.

File: code/modules/mob/living/carbon/brain/brain_item.dm [hunk 1]
```diff
@@ -98,7 +98,7 @@
 	icon_state = "green slime extract"
 
 /obj/item/organ/brain/golem
-	name = "chem"
```
Comment (NanakoAC) [inline code/modules/mob/living/carbon/brain/brain_item.dm]: wtf is a chelm

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -888,6 +888,32 @@
...
+	playsound(loc, 'sound/items/countdown.ogg', 125, 1)
+	sleep(20)
+	playsound(loc, 'sound/effects/alert.ogg', 125, 1)
+	sleep(10)
+	var/turf/O = get_turf(src)
+	var/fragger = rand(50,100)
+	density = 0
+	explosion(O, -1, -1, 2)
+	var/list/target_turfs = getcircle(O, 7)
+	var/fragments_per_projectile = round(fragger/target_turfs.len)
```
Comment (NanakoAC) [inline code/modules/mob/living/silicon/robot/robot.dm]: I think it'd be a better idea to create a reuseable effect object to fire shrapnel all around

Having snowflake code in everything that should throw shrapnel doesn't seem like a good approach

File: code/modules/mob/living/carbon/brain/brain_item.dm (hunk 1 above)
Comment (Alberyk) [inline code/modules/mob/living/carbon/brain/brain_item.dm]: http://www.jewishgen.org/yizkor/chelm/che045.html

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 2 above)
Comment (LordFowl) [inline code/modules/mob/living/silicon/robot/robot.dm]: I have no interest in making things create more shrapnel. I can't really think of any cases except cyborgs and grenades.

**Pull request merged**